content/en/docs/topics/advanced.md

```diff
@@ -143,3 +143,68 @@ func main() {
 }
 
 ```
+
+## Storage backends
+
+By default, Helm stores release information in Secrets in the namespace of
+the release.
+
+### ConfigMap storage backend
+
+To enable the ConfigMap backend, you'll need to set the environmental variable
+`HELM_DRIVER` to `configmap`.
+
+You can set it in a shell as follows:
+
+```shell
+export HELM_DRIVER=configmap
+```
+
+If you want to switch from the default backend to the ConfigMap
+backend, you'll have to do the migration for this on your own. You can retrieve
+release information with the following command:
+
+```shell
+kubectl get secret --all-namespaces -l "owner=helm"
+```
+
+**PRODUCTION NOTES**: It is not recommended to use ConfigMaps for sensitive data as
+it is stored unencoded. Secrets data is stored base64-encoded.
+
+### SQL storage backend
+
+There is a beta SQL storage backend that stores release information in an SQL database.
```

Comment:
The [pull request for the feature](https://github.com/helm/helm/pull/7635) does not note it as "beta". Do we have more than supported or experimental? Should this be noted as "beta" if we don't really have that designation?